test/parallel/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const tls = require('tls');
+
+const sslcontext = tls.createSecureContext({
+  cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
+  key: fs.readFileSync(common.fixturesDir + '/test_key.pem'),
```

Comment:
Yes, this simplify diff when if third line added to a list.

http://ecma262-5.com/ELS5_HTML.htm#Section_11.1.5